scanpy_scripts/scanpy_filter_genes.py

```diff
@@ -22,7 +22,11 @@ def main(argv=None):
     adata = read_input_object(args.input_object_file, args.input_format)
 
     if args.subset_list:
-        adata = adata[:, adata.var_names.isin(args.subset_list)]
+        k = adata.var.index.isin(args.subset_list)
+        if sum(k) == 0:
```

Comment:
Silly question- why a sum for k rather than a length?

Reply:
Because k is an array of boolean values.

Reply:
Ahh cool- thanks